src/main/java/com/google/api/codegen/transformer/ApiMethodTransformer.java

```diff
@@ -238,6 +240,12 @@ public StaticLangApiMethodView generateCallableMethod(MethodTransformerContext c
         !ServiceMessages.s_isEmptyType(context.getMethod().getOutputType()));
     methodViewBuilder.isPageStreaming(false);
 
+    MethodConfig methodConfig = context.getMethodConfig();
+    if (methodConfig.isGrpcStreaming()) {
+      GrpcStreamingConfig grpcStreaming = methodConfig.getGrpcStreaming();
+      methodViewBuilder.streamingType(grpcStreaming.getType());
+    }
```

Comment:
I am not sure if I understand this comment. I moved this if block into the `MethodConfig` class.